TROUBLESHOOTING.md

```diff
@@ -160,6 +160,36 @@ ruby-lsp
 Is there any extra information given from booting the server manually? Or does it only fail when booting through the
 extension?
 
+### Outdated Version
```

Comment:
This is under `Diagnosing the Problem`; would it make sense to move that to under `Common Problems`?